src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &-star {
+    width: 16px;
+    height: 16px;
+    left: 80px;
```

Comment:
The `left` property will have no effect unless the element has a positioning context. Consider adding `position: relative;` or another positioning property if needed.